src/language/dafnyInstallation.ts

```diff
@@ -123,6 +129,32 @@ export class DafnyInstaller {
     }
   }
 
+  private async installFromSource() {
+    const { execSync } = require('child_process');
```

Comment:
Don't use `require` within ES Modules. As well as avoid using blocking/sync functions whenever possible. You may want to have a look here:
https://github.com/dafny-lang/ide-vscode/blob/cf35f2c4539bb75d102f7219ef185d4bdd5b0ff7/src/dotnet.ts#L15

It should help you solve both things, making an asynchronous call and using the `import` statement.